Packs/CommonTypes/IncidentFields/incidentfield-Affected_Hosts.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "affectedhosts",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_affectedhosts",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Affected Hosts",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "tagsSelect",
+    "unmapped": false,
+    "unsearchable": true,
```

Comment:
This should be searchable IMO. Maybe related to the reason that the dashboard doesn't work with them